src/order/lattice.lean

```diff
@@ -710,6 +710,25 @@ instance linear_order.to_distrib_lattice {α : Type u} [o : linear_order α] :
 instance nat.distrib_lattice : distrib_lattice ℕ :=
 by apply_instance
 
+/-! ### Dual order -/
+
+open order_dual
+
+@[simp] lemma of_dual_inf [has_sup α] (a b) : of_dual (a ⊓ b) = (of_dual a : α) ⊔ of_dual b := rfl
+@[simp] lemma of_dual_sup [has_inf α] (a b) : of_dual (a ⊔ b) = (of_dual a : α) ⊓ of_dual b := rfl
```

Comment:
It probably results in line-wrapping, but I think it's better to be explicit here (and below for `linear_order`)
```suggestion
@[simp] lemma of_dual_inf [has_sup α] (a b : order_dual α) : of_dual (a ⊓ b) = of_dual a ⊔ of_dual b := rfl
@[simp] lemma of_dual_sup [has_inf α] (a b : order_dual α) : of_dual (a ⊔ b) = of_dual a ⊓ of_dual b := rfl
```

Reply:
Yeah, I avoided this specifically because of line-wrap.

Reply:
I don't think that's a particulary good justification; your spelling makes it look like some weird coercion is going on.

Perhaps what you really want is a notation for `order_dual`?

Reply:
That would solve it, indeed. Wikipedia mentions `ᵈ` and `ᵒᵖ`, which are both not very suitable. What about something like `ᵒᵈ`?

Reply:
`α⤩` might also make sense